.circleci/initialize.py

```diff
@@ -0,0 +1,358 @@
+    print("No redis/postgres databases to add")
+
+if len(linkedServices.items()) == 1:
+    print(
+        "Linking database...",
+    )
+elif len(linkedServices.items()) > 1:
+    print(
+        "Linking databases...",
+    )
+    for serviceType in linkedServices:
+        query_linkService = gql(
+            """
+            mutation (
+                $appname: String
+                $serviceType: ServiceType=redis,
+                $serviceName: String
+            ) {
+                linkService (
+                    appname: $appname,
+                    serviceName: $serviceName,
+                    serviceType: $serviceType
+                ) {
+                    error
+                }
+            }
+            """
+        )
+        params_linkService = {
+            "appname": APPNAME,
+            "serviceName": serviceName,
+            "serviceType": serviceType,
+        }
+
+        sleep(5)
+        result = client_service.execute(
+            query_linkService, variable_values=params_linkService
+        )
+        handle_error(result, accepted_errors)
+
+        print(
+            ". {serviceName}, {serviceType}".format(
+                serviceName=serviceName, serviceType=serviceType
+            )
+        )
+else:
+    print("No redis/postgres databases to link")
+
+if len(mounts.items()) != 0:
+    print("Mapping directories...")
+    for host_dir, target_dir in mounts.items():
+        query = gql(
+            """
+            mutation (
+                $hostDir: String,
+                $targetDir: String,
+                $appname: String
+            ) {
+                mountDirectory(
+                    hostDir: $hostDir,
+                    targetDir: $targetDir,
+                    appname: $appname
+                ) {
+                    error
+                }
+            }
+            """
+        )
+        params = {
+            "hostDir": host_dir,
+            "targetDir": target_dir,
+            "appname": APPNAME,
+        }
+
+        result = client_service.execute(query, variable_values=params)
+        handle_error(result, accepted_errors)
+
+        print(
+            "  Mapping hostDir: {host_dir} to targetDir: {target_dir}".format(
+                host_dir=host_dir, target_dir=target_dir
+            )
+        )
+else:
+    print("No directories to map")
+
+if len(environmentVariables.items()) != 0:
+    print("Adding environment variables...")
+    environmentVariables_filter = tuple(
+        # These environment variables are created automatically by Dash Enterprise and
+        # do not need to be manually modified.
+        [
+            "DOKKU",
+            "DASH",
+            "DATABASE_URL",
+            "GIT_REV",
+            "REDIS_URL",
+            "SCRIPT_NAME",
+            "NO_VHOST",
+        ]
+    )
+    for envar_name, envar_value in environmentVariables.items():
+        if envar_name.startswith(environmentVariables_filter) is not True:
```

Comment:
Can also be written:
```suggestion
        if not envar_name.startswith(environmentVariables_filter):
```